Tiltfile

```diff
@@ -0,0 +1,93 @@
+load("ext://restart_process", "docker_build_with_restart")
+load("ext://helm_resource", "helm_resource", "helm_repo")
+load("ext://configmap", "configmap_create")
+
+# A list of directories where changes trigger a hot-reload of PATH.
+# Note: this list needs to be updated each time a new package is added to the repo.
+hot_reload_dirs = ["cmd", "config", "gateway", "health", "message", "qos", "relayer", "request", "router"]
+
+# Load the existing config file, if it exists, or use an empty dict as fallback
+localnet_config_path = "localnet_config.yaml"
+localnet_config = read_yaml(localnet_config_path, default={})
+
+# TODO_UPNEXT(@adshmh): Package the default Helm chart for PATH and upload the pokt-network repo.
```

Comment:
What is the `pokt-network` repo and what does it have to do here?

Reply:
I was also wondering this. When playing with this locally I was only able to get this to run when removing the references to the helm repo here.

In the interests of having the Tilt config be functional from the very first PR and onwards, WDYT about removing anything that is not yet ready for use and/or makes the Tiltfile non-functional for now (with the understanding we will put it back if/when ready and needed)?